src/apps/shared/commands/patches/m2_6733_add_answer_input_subjects.py

```diff
@@ -0,0 +1,50 @@
+import uuid
+
+from rich import print
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from apps.workspaces.service.workspace import WorkspaceService
+from infrastructure.database import session_manager
+
+
+async def update_answers(session):
+    query = """
+    update answers
+    set input_subject_id = source_subject_id::uuid
```

Comment:
Yes makes sense that this is `source_subject_id` as long as this is part of MI R1.